components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -564,7 +564,7 @@ const HighlightInputForm = ({ refreshCallback }: HighlightInputFormProps): JSX.E
                 </span>
               )}
             </p>
-            <div className="flex flex-col gap-2 p-2 text-sm bg-white border rounded-lg">
+            <div className="flex flex-col gap-2 p-2 text-sm bg-white border rounded-lg outline-1 focus-within:outline focus-within:outline-orange-500">
```

Comment:
Textareas in general need default focus states like some other components, but it's fine to do it on an individual level for the moment.

These are the styles we currently use for focus on other form inputs.

```suggestion
            <div className="flex flex-col gap-2 p-2 text-sm bg-white border rounded-lg focus-within:outline-none focus-within:ring  focus-within:border-orange-500 focus-within:ring-orange-100">
```